subprojects/gst-plugins-bad/sys/msdk/gstmsdkenc.c

```diff
@@ -1384,12 +1441,10 @@ gst_msdkenc_create_buffer_pool (GstMsdkEnc * thiz, GstCaps * caps,
 #ifndef _WIN32
   pool = gst_msdk_create_va_pool (thiz, caps, num_buffers);
 #else
-  /* Currently use system pool for windows path */
+  pool = gst_msdk_create_d3d11_pool (thiz, num_buffers);
+#endif
   if (!thiz->use_video_memory)
```

Comment:
It looks like both linux and windows path set thiz->use_video_memory as TRUE, so we will not use the system pool for encoder, right?

Reply:
Currently yes. I think we can add another patch to remove this use_video_memory after all the refactor work is done.